webpack.playground.config.js

```diff
@@ -55,8 +54,30 @@ module.exports = {
       // webpack-dev-server v2 only watch contentbase on root level
       // explicitly list folders to watch for browser auto reload
       // sub-folders can be removed when upgrade to webpack-dev-server v3
-      `${TARGET}/js`,
-      `${TARGET}/css`,
+      path.join(TARGET, 'js'),
```

Comment:
this is too verbose.. I like helper better. we can add comment for explanation.

Reply:
@pradeepdewda-okta turns out this won't update the mock server anyway.
I'm going to cancel this PR and re-submit with a fix.